overcloud.yml

```diff
@@ -27,6 +27,27 @@
           src: "firstboot-nvme.yaml.j2"
           dest: "/home/stack/firstboot-nvme.yaml"
         when: nvme is defined
+      - block:
+          - name: adjust max number of concurrent builds to default (override infrared)
+            ini_file:
+                path: /var/lib/config-data/puppet-generated/nova/etc/nova/nova.conf
+                section: DEFAULT
+                option: max_concurrent_builds
+                value: 10
+            become: yes
+
+          - name:  get nova services
+            command: systemctl list-units | grep 'nova.*.service'
```

Comment:
you can use awk in the above command as it is currently not just fetching the name of the nova services instead it registering 
```
[stack@undercloud ~]$ systemctl list-units --all | grep 'nova.*.service'
  openstack-nova-api.service                                                                                     loaded    active   running   OpenStack Nova API Server
  openstack-nova-compute.service                                                                                 loaded    active   running   OpenStack Nova Compute Server
  openstack-nova-conductor.service                                                                               loaded    active   running   OpenStack Nova Conductor Server
  openstack-nova-scheduler.service                                                                               loaded    active   running   OpenStack Nova Scheduler Server
```
So the CI job is failing

Reply:
My bad, I missed the awk...